asyncapi/github/dispatcher_service.bal

```diff
@@ -43,36 +39,24 @@ service class DispatcherService {
         _ = self.services.remove(serviceType);
     }
 
-    // We are not using the (@http:payload GenericEventWrapperEvent g) notation because of a bug in Ballerina.
-    // Issue: https://github.com/ballerina-platform/ballerina-lang/issues/32859
-    resource function post .(http:Caller caller, http:Request request) returns http:Response|error? {
-        json payload = check <@untainted> request.getJsonPayload();
-        byte [] binaryPay = <@untainted>  payload.toString().toBytes();
-        string eventName = check  request.getHeader("X-GitHub-Event");
-        if (request.hasHeader("X-Hub-Signature-256")) {
-            string secret = check  request.getHeader("X-Hub-Signature-256");
-            string trimmedSecret = secret.substring(7, secret.length());
-            byte [] output = check crypto:hmacSha256(binaryPay, self.listenerConfig.webhookSecret.toBytes());
-            string computedDigest = output.toBase16();
-            if (trimmedSecret.length() != computedDigest.length() || trimmedSecret !== computedDigest) {
-                // Validate secret with X-Hub-Signature-256 header for intent verification
-                log:printError("Signature verification failure");
-                http:Response response = new;
-                response.statusCode = http:STATUS_UNAUTHORIZED;
-                response.setPayload("Signature verification failure");
-                return response;
-            }
+    remote function onEventNotification(websub:ContentDistributionMessage event)
+    returns Acknowledgement|websub:SubscriptionDeletedError|error? {
```

Comment:
Let's align the return statement with the opening bracket of the function.

Reply:
Updated in https://github.com/ballerina-platform/asyncapi-triggers/commit/e0c56638e7d25694ff986c19c894b01fc776abb8